data/migrations/20190909104944_alter-shoutouts-table.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex) {
+  return knex.schema.table('shoutouts', tbl => {
```

Comment:
Awesome man, I think we already have a migration file for altering the shoutout table to add the message timestamp, maybe we can delete this one